docs/codeql/codeql-language-guides/customizing-library-models-for-actions.rst

```diff
@@ -0,0 +1,46 @@
+.. _customizing-library-models-for-actions:
+
+Customizing Library Models for GitHub Actions
+=========================================
+
+.. include:: ../reusables/beta-note-customizing-library-models.rst
+
+GitHub Actions analysis can be customized by adding library models in data extension files.
```

Comment:
Would it make sense to link to the main documentation page for data extensions? Or maybe to include links to some examples in the code.

Reply:
I think so but I would rather do it separately for all the languages.